backend/project_alpha/settings.py

```diff
@@ -104,6 +105,12 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.1/howto/static-files/
 
+UPLOAD_IMAGE_MAX_SIZE_MB = 2.0
```

Comment:
please move UPLOAD_IMAGE_MAX_SIZE_MB and ALLOWED_IMAGE_FORMATS to `.yaml` configs that we already have.